subprojects/docs/src/docs/userguide/migration/upgrading_version_7.adoc

```diff
@@ -267,3 +267,8 @@ configurations {
 }
 ```
 
+
+[[ear_convention_deprecation]]
+=== Deprecated `project-report` plugin conventions
+
+link:{groovyDslPath}/org.gradle.api.plugins.ProjectReportsPluginConvention.html[ProjectReportsPluginConvention] is now deprecated and scheduled for removal in Gradle 8.0. Clients should configure the project report tasks directly. Also, link:{javadocPath}/org/gradle/api/DomainObjectCollection.html#withType-java.lang.Class-[tasks.withType(War.class).configureEach(...)] can be used to configure each task of the same type.
```

Comment:
```suggestion
link:{groovyDslPath}/org.gradle.api.plugins.ProjectReportsPluginConvention.html[ProjectReportsPluginConvention] is now deprecated and scheduled for removal in Gradle 8.0. Clients should configure the project report tasks directly. Also, link:{javadocPath}/org/gradle/api/DomainObjectCollection.html#withType-java.lang.Class-[tasks.withType(War.class).configureEach(...)] can be used to configure each task of the same type.
```

What type should be used here (other than War)?

Reply:
Fixed